implementations/rust/ockam/ockam_api/tests/auth.rs

```diff
@@ -69,7 +69,7 @@ async fn credential(ctx: &mut Context) -> Result<()> {
 
     // Add the member via the enroller's connection:
     let mut c = direct::Client::new(route![e2a, "auth"], ctx).await?;
-    c.add_member(IdentityId::new(m.identifier()?.key_id()))
+    c.add_member(IdentityId::new(m.identifier().key_id()))
```

Comment:
Does it still need to access `key_id`?

Reply:
This part will change in future PRs soon. I limited the scope of this one.